automatic/7zip.install/update.ps1

```diff
@@ -0,0 +1,29 @@
+. "$PSScriptRoot\..\7zip\update.ps1"
+
+$softwareNamePrefix = '7-zip'
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -FileNameBase '7zip'
+  $Latest.ChecksumType = 'sha256'
+}
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*softwareName\s*=\s*)'.*'" = "`$1'$softwareNamePrefix $($Latest.RemoteVersion)*'"
+    }
+    ".\tools\chocolateyUninstall.ps1" = @{
+      "(?i)(\s*\-SoftwareName\s+)'.*'" = "`$1'$softwareNamePrefix $($Latest.RemoteVersion)*'"
```

Comment:
no, $Latest.Version will be changed if there is a fix version, which will make the uninstall script unusable since it won't be able to find the uninstall key